core/graph/README.md

```diff
@@ -54,4 +54,104 @@ to be run before and after an operation chain is executed.
 
 You can use hooks to do things like custom logging or special operation
 chain authorisation. To implement your own hook, just implement the `GraphHook`
-interface and register it with the graph when you build a `Graph` instance.
\ No newline at end of file
+interface and register it with the graph when you build a `Graph` instance.
+
+## FAQs
+To use an example of the authorised Graph exporter within the road-traffic-demo using the proxy-store, follow these steps.
+
+-Clone Gaffer twice, instance A and instance B.

```

Comment:
I think we could actually demonstrate the export using just a single instance of accumulo and just export to a different Accumulo table (using a different graphId but the same properties and schema).